CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.0 / 2021-10-19
+
+* [FEATURE] Add support for security-related HTTP Headers. #41
```

Comment:
Don't think "Headers" needs to be capitalized https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

```suggestion
* [FEATURE] Add support for security-related HTTP headers. #41
```

Reply:
updated